satpy/etc/readers/seviri_l2_bufr.yaml

```diff
@@ -1184,10 +1274,11 @@ datasets:
 
   pressure:
     name: pressure
+    long_name: Wind Pressure
```

Comment:
I would suggest to change to  "Air Pressure at Wind Level"

Reply:
done